L/LinearFold/build_tarballs.jl

```diff
@@ -0,0 +1,41 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "LinearFold"
+version = v"1.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/LinearFold/LinearFold/archive/refs/tags/v$(version.major).$(version.minor).tar.gz",
+                  "2ae56b5f183472c2de96782e770a91e57f82e0ab511dfc0d9d612aa4e6155f60"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/LinearFold-*/
+
+make -j${nproc}
```

Comment:
Looks like the [upstream `Makefile`](https://github.com/LinearFold/LinearFold/blob/f985f1dfd4e90912fad16780cb7337d618242a3d/Makefile#L8) sets
```makefile
CC=g++
```
(yes, `CC` instead of `CXX`, sigh) which is wrong for BSD systems
```suggestion
make -j${nproc} CC=${CXX}
```